crypto/store/store_local.h

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
I don't really care too much whether we standardise on "_local.h", "locl.h" or "lcl.h" - but having 3 different variants is just confusing. I favour "locl.h" as the path of least resistance and the closest to a standard that we have, but if someone wants to go to the trouble of renaming all the files then so be it.